### DISCUSSION TITLE
Yandex Cloud Deploy

### DISCUSSION THREAD

**Discussion by Jhon-Mosk** [anuraghazra/github-readme-stats#2871] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

Добрый день,

доработал ваш проект для разворачивания в облаке Яндекса, используя serveless functions и Api gateway. На данный момент использование этого решения не требует денег. Возможно стоит написать инструкцию и выложить моё решения для сообщества?

**Comment by rickstaa** [comment, marked as answer]
@Jhon-Mosk I tried translating your text, but Google didn't return something I could understand 🤔. I think you want to deploy on Yandex cloud. Although we do not support platforms other than Vercel it is possible. Please follow https://github.com/anuraghazra/github-readme-stats#on-other-platforms.

**Reply by Jhon-Mosk** [reply]
I have already deployed in the Yandex cloud. Perhaps my solution will be useful to the community?

**Reply by rickstaa** [reply]
Of course 🚀. Feel free to add your solution here or create a PR to the README if it differs from the one we already have in there 👍🏻.

**Reply by Jhon-Mosk** [reply]
My solution is free. You don't have to pay for cloud services. I rewrote the project on node.js, wrote an API for it, launched it in a cloud function. I can share the code. Tell anyone how to deploy a project in the Yandex cloud. The solution is also suitable for the Google cloud

**Reply by rickstaa** [reply]
You can add your solution here if you like 👍🏻. Please bear in mind that all the answers shared in the README (i.e. Vercel and express.js) are also free.

**Reply by rickstaa** [reply]
You should be able to deploy GRS without any changes on any cloud using the steps provided at https://github.com/anuraghazra/github-readme-stats#on-other-platforms 🤔.

**Reply by Jhon-Mosk** [reply]
Make pull request
![image](https://github.com/anuraghazra/github-readme-stats/assets/40711760/49e2426e-10a1-48a0-aade-a770a7b9505c)
What next?

**Reply by rickstaa** [reply]
@Jhon-Mosk, thanks for the pull request. I checked it, and since it contains many code changes, merging #2872 into the upstream is not wise. It can be valuable for people who want to deploy on Yandex. Neverteless, I would think that fewer changes would be needed to run it on Yandex cloud since, for most clouds, https://github.com/anuraghazra/github-readme-stats#on-other-platforms was sufficient, but it could be because of how the Yandex cloud functions work.

**Reply by Jhon-Mosk** [reply]
Fine. Wouldn't it be a violation if I put this code in my repository? Do I need to list Anura Ghazra in package.json? Do I need to attach the MIT license with his name?

**Reply by rickstaa** [reply]
I'm not a lawyer, but as explained in the license, you can do anything with our code. It simply protects us from having to give a warranty and having liability (see https://github.com/anuraghazra/github-readme-stats/blob/master/LICENSE). As far as I know, it does not require you to use the same license but to include the original license in your new project. The old code you copied falls under the old right, and the new regulation is under the new license you added. Apart from that, it is common practice in OS projects to reference the original project in the README (see https://github.com/anuraghazra/github-readme-stats#sparkling_heart-support-the-project).

**Reply by Jhon-Mosk** [reply]
Thank you
